x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/TumblingWindow.java

```diff
@@ -47,8 +49,23 @@
  */
 public class TumblingWindow implements Executable {
 
+    private static final int CACHE_MAX_SIZE = 63;
+
     private final Logger log = LogManager.getLogger(TumblingWindow.class);
 
+    /**
+     * Simple cache for removing duplicate strings (such as index name or common keys).
+     * Designed to be low-effort and thus optimistic in nature.
+     * Thus it has a small, upper limit so that it doesn't require any cleaning up.
```

Comment:
Since we have a `Cache` class in the common lib, which is more complex and supports concurrency, could we have a comment here that concurrency is not needed?